docker_setup_scripts/docker_setup_scripts_common.sh

```diff
@@ -85,6 +85,15 @@ yb_end_group() {
   echo "::endgroup::"
 }
 
+yb_apt_add_packages() {
+  # Add Bazel package
+  apt-get install -y apt-transport-https curl gnupg
+  curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
+  mv bazel.gpg /etc/apt/trusted.gpg.d/
+  echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
```

Comment:
Please use `echo >... "..."` or `echo "..." >...` syntax for consistency. Pipe is not required here.